miden-crypto/src/rand/rpo.rs

```diff
@@ -79,21 +79,24 @@ impl RpoRandomCoin {
         self.current += 1;
         self.state[self.current - 1]
     }
-}
 
```

Comment:
I think we will no longer need RandomCoin-s but maybe it is worth keeping them for some transition period. cc @bobbinth

Reply:
Tracked in #725